source/grid/tria.cc

```diff
@@ -11898,6 +11898,9 @@ void Triangulation<dim, spacedim>::create_triangulation(
               cell->face(pair.first)->set_boundary_id(pair.second);
         }
     }
+
+  // inform all listeners that the triangulation has been created
+  signals.create();
```

Comment:
This is an unrelated change. This PR has been sitting around for too long already, but could you add a changelog entry for this, along with a test?

Reply:
...after I merge, that is.

Reply:
The tests `simplex/step-55` and `simplex/step-67` fail without this line of code.

I've provided a changelog in #14866. Do you want me to add another test?

Reply:
But if these tests fail without this patch, how did they succeed before? Or are you saying that moving the active-fe-index check exposed the bug, and consequently these tests failed and required the calling of the signal?

If so, no, no further tests necessary!

Reply:
> Or are you saying that moving the active-fe-index check exposed the bug, and consequently these tests failed and required the calling of the signal?

Exactly this was the case. Moving the check exposed the bug. See also https://github.com/dealii/dealii/pull/14866#issuecomment-1463608649

Reply:
OK, thanks for confirming!